action.yml

```diff
@@ -18,6 +18,13 @@ inputs:
       Whitespace-separated fully-qualified image names.
       Each tag must be in the format: <registry>/[<namespace>/]<repository>:<tag>
     required: true
+  
+  smoke_test:
+    description: > 
+      Bash shell script to run as a smoke test against the built image.
+      Inside the script, you will have access to a variable called 'IMAGE_NAME' 
+      which you can use in your test to run the image.
```

Comment:
We should add a note here that the script is _currently_ only run against the `linux/amd64` version of the image, and that it will have access to a local docker daemon.

However, users should try to write tests that don't rely on the arch being `amd64` nor on the daemon being local, as these are things we may want to change later to enable testing all the other non-amd64 images.

Reply:
We should also note that the script must exit non-zero in order to mark the test as failed.